src/Render/VFS/RelativeFile.cs

```diff
@@ -8,12 +8,14 @@ internal sealed class RelativeFile {
 
 		public FileInfo Source { get; }
 		public FileInfo Destination { get; }
-		public DocumentContext Context { get; }
+		public DocumentContext Context { get; }
+		public string SourceEditLink { get; }
 
-		public RelativeFile( DocumentContext context, FileInfo source, FileInfo destination ) {
+		public RelativeFile( DocumentContext context, FileInfo source, FileInfo destination, string sourceEditLink ) {

```

Comment:
nit: consider passing it along as a `Uri` object to maintain the type safety until it's used

Reply:
Done.